setup.py

```diff
@@ -11,24 +9,20 @@
 
 
 requires = [
-    'requests',
+    'requests>=2.6.0',
 ]
 
-version = ''
-with open('telex/__init__.py', 'r') as fd:
-    version = re.search(r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]',
-                        fd.read(), re.MULTILINE).group(1)
-
-with open('README.md', 'r', 'utf-8') as fd:
-    readme = fd.read()
-
+with open('README.md') as f:
+    readme = f.read()
+with open('LICENSE') as f:
```

Comment:
We don't want to read the text of the license into memory and shove it into our package metadata. That will break rendering of the README if we upload to PyPI.